core/dbt/events/types.py

```diff
@@ -715,6 +715,17 @@ def message(self) -> str:
         return line_wrap_message(deprecation_tag(description))
 
 
+class ModulesItertoolsUsageDeprecation(WarnLevel):
+    def code(self) -> str:
+        return "D034"
```

Comment:
Do we know why we are missing these middle Numbers since D035 is already taken?  Not a blocker, just curious.

Reply:
I had added the protos but just haven't gotten around to the implementation of 034 and 033 in particular! Getting to them now :)